content/pages/privacy.md

```diff
@@ -1,7 +0,0 @@
-Title: Privacy Policy
-URL: privacy.html
-save_as: privacy.html
-template: lucene/tlp/page
-
```

Comment:
Alternatively we could retain this page and on it add a link to https://privacy.apache.org/policies/privacy-policy-public.html from here?

Reply:
personally I like the PR the way you have it. less to maintain.